demo/full/scripts/controllers/Player.jsx

```diff
@@ -33,15 +36,65 @@ class Player extends React.Component {
     this._$destroySubject = new Subject();
     this._$destroySubject.subscribe(() => player.destroy());
 
-    player.$get("isSeeking", "isBuffering", "isLoading", "isReloading", "isStopped")
+    player.$get("isSeeking",
+                "isBuffering",
+                "isLoading",
+                "isReloading",
+                "isStopped",
+                "liveGap",
+                "lowLatencyMode"
+    )
       .pipe(takeUntil(this._$destroySubject))
       .subscribe(([
         isSeeking,
         isBuffering,
         isLoading,
         isReloading,
         isStopped,
+        liveGap,
+        lowLatencyMode,
       ]) => {
+        const { isCatchingUp,
+                lastCatchUpRate,
```

Comment:
I don't get why this one (`lastCatchUpRate`) is needed. We should just be able to read the current playback rate already in the player state.